OpenTrons/opentrons-api
# feat: Opentrons ai client review

# Overview

This PR handles the Submit Prompt action.

![image](https://github.com/user-attachments/assets/d3b52b9f-44b7-47f6-8bb8-5b92a6aacfe6)

![image](https://github.com/user-attachments/assets/2aaf26cd-6162-4fbe-b433-d2b30659fe6d)


## Test Plan and Hands on Testing

- On the landing page click Create a new protocol button, you will be redirected to the new page
- fill up the required information in all sections and click Confirm
- Submit Prompt button is enabled, click the button
- User is redirected to Chat page with the filled up prompt.
- [WIP] Prompt should be submitted already, this will be done in a separate PR

## Changelog

 - Add Submit Prompt handling and small refactors

## Review requests

- Verify submitting prompt.

## Risk assessment

- low
